src/js/chartAuthoring.js

```diff
@@ -237,6 +334,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onDataAppliedToDataEntryPanel.fire();
+    };
+
+    floe.chartAuthoring.addSonificationControlsHandlers = function(that) {
+        var playButton = that.locate("sonifierPlay"),
+            stopButton = that.locate("sonifierStop");
+
+        playButton.click(function(e) {
```

Comment:
Consult @jobara on what we currently have for a declarative event binding system. If we can't locate something suitable, we can import the following sketch:

https://gist.github.com/amb26/e25255f43d36d08a8943